frontend/src/components/auth/AuthForm.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { Field, reduxForm } from 'redux-form'
+
+const AuthForm = (props) => {
+  const { handleSubmit, pristine, reset, submitting, serverLogin, history, auth } = props
+
+  const submit = (values) => {
+    serverLogin(values)
+      .then(res => {
+        if (res.payload) {
+          const session = res.payload.data
+          setCookie(session.UserID, session.SessionID, session.Expires)
+          history.push('/accounts')
+        }
+      })
+        
+    const setCookie = (UserID, SessionID, Expires) => {
```

Comment:
It would be more readable and easy for extraction if you move this function on the same level as AuthForm.